lib/functions.js

```diff
@@ -0,0 +1,117 @@
+const originPrivateKey = "01009280BDB84B8F8AEDBA205FE3552689964A5626EE2C60AA10E3BF22A91A036009"
+const archethic = require('archethic')
+const yesno = require('yesno')
+const chalk = require('chalk')
+const crypto = require('crypto')
+const fs = require('fs')
+const path = require('path')
+
+module.exports = {
```

Comment:
I think you should create multiple files about file management, transaction building grouped by domain